### PR TITLE
Load templates as memmap rather than in memory

### PR DESCRIPTION
**Pull request by DradeAW**
When generating very big dataset, it's very easy to have templates bigger than 60 GiB

This PR makes the templates be loaded as memmaps rather than in memory .

### CODE REVIEW

**Comment by DradeAW** [comment]
Ok I can confirm that generating the templates now works while using a lot less RAM

Tomorrow I'll check whether there is a conversion to memory while generating the recording

**Comment by DradeAW** [comment]
I need to test this last commit,
@alejoe91 Do you think they're good or should I also change something else?

**Review by alejoe91 — APPROVED** [review]

**Comment by alejoe91** [comment]
Looks good to me! Let me know when I should merge

File: src/MEArec/tools.py [hunk 1]
```diff
@@ -565,6 +572,8 @@ def save_recording_generator(recgen, filename=None, verbose=False):
         Path to .h5 file
     verbose : bool
         If True output is verbose
+    include_spike_traces: bool, default=True
+        If True, will include the spike traces (can be heavy)
```
Comment (alejoe91) [inline src/MEArec/tools.py:576]:
```suggestion
        If True, will include the spike traces (which can be large for many units)
```

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**